sdk/test-utils/recorder/src/utils/utils.ts

```diff
@@ -363,3 +363,10 @@ export function assertEnvironmentVariable(variable: string): string {
   if (!value) throw new Error(`${variable} is not defined`);
   return value;
 }
+
+/**
+ * Polling options that don't wait in playback mode.
+ */
+export const testPollingOptions = {
+  updateIntervalInMs: isPlaybackMode() ? 0 : undefined,
```

Comment:
NIT
we can import the type from core-lro, so that they are in sync always.

Reply:
core-lro's LroEngineOptions has a different property `intervalInMs`, which means some other packages needs a different testing option...

Reply:
we can add `intervalInMs: isPlaybackMode() ? 0 : undefined,` here too when needed in the future, it should be compatible with both.